examples/2-handle-prompt/index.js

```diff
@@ -0,0 +1,111 @@
+import { createServer } from 'node:http';
+import { Octokit } from "@octokit";
+import { 
+    createAckEvent, 
+    createDoneEvent, 
+    createTextEvent, 
+    parseRequestBody, 
+    prompt,
+    verifyRequestByKeyId
+ } from "@copilot-extensions/preview-sdk";
+
+
+// Define the handler function
+const handler = async (request, response) => {
+
+    if (request.method !== 'POST') {
+        // Handle other request methods if necessary
+        response.writeHead(405, { 'Content-Type': 'text/plain' });
+        console.log(`Method ${request.method} not allowed`);
+        
+        response.end('Method Not Allowed');
+    }
+
+    console.log('Received POST request');
+
+    // get a token to use
+    const tokenForUser = request.headers['x-github-token'];
+    
+    // get the user information with the token        
+    const octokit = new Octokit({ auth: tokenForUser });
+    const user = await octokit.request("GET /user");
+    const userHandle = user.data.login
+    
+    // Collect incoming data chunks to use in the `on("end")` event
+    let body = '';
+    request.on('data', chunk => {
+        body += chunk.toString();
+    });
+
+    // Parse the collected data once the request ends
+    request.on('end', async () => {
+        try {
+            const payloadIsVerified = await verifyRequestByKeyId(
+                request.body,
+                signature,
+                keyId,
+                {
+                  token: process.env.GITHUB_TOKEN,
+                },
+            );
+
+            if (!payloadIsVerified) {
+                console.error('Request verification failed');
+                response.writeHead(401, { 'Content-Type': 'text/plain' });
+                response.end('Request could not be verified');
+                return;
+            }
+
+            // start sending the response back to Copilot
+            console.log('Handling response');
+            // write the acknowledge event to let Copilot know we are handling the request
+            // this will also show the message "Copilot is responding" in the chat
+            response.write(createAckEvent());
+
+            // parse the incoming body as that has the information we need to handle the request / user prompt
+            const payload = parseRequestBody(body);
+            
+            // start writing text in the response
+            const textEvent = createTextEvent(`Hello ${userHandle}, \n\n`);
+            response.write(textEvent);
+            // add new lines to mark the difference between the fixed text and the dynamic text
+            response.write("\n\n");
```

Comment:
can you explain what you mean by

> mark the difference between the fixed text and the dynamic text

None of the `\n` are necessary
```suggestion
            const textEvent = createTextEvent(`Hello ${userHandle}`);
            response.write(textEvent);
```